fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/api/IDatabaseTypeAdapter.java

```diff
@@ -27,6 +27,14 @@
      */
     public String blobClause(long size, int inlineSize);
 
+    /**
+     * Generate a clause for double data type
+     * @return
+     */
+    default public String doubleClause() {
```

Comment:
Default is recommended to come second... 

https://stackoverflow.com/a/16732059/1873438

Reply:
good to know this, was just thinking to align with existing codes. let me fix them

Reply:
I saw this warning in SonarLint

Reply:
done